Lib/argparse.py

```diff
@@ -1207,8 +1207,9 @@ def __call__(self, string):
             return open(string, self._mode, self._bufsize, self._encoding,
                         self._errors)
         except OSError as e:
-            message = _("can't open '%s': %s")
-            raise ArgumentTypeError(message % (string, e))
+            args = {'filename': string, 'error': e}
+            message = _("can't open '%(filename)s': %(error)s")
+            raise ArgumentTypeError(message % (args))
```

Comment:
Please remove the brackets around `args`

Reply:
Thanks, fixed